src/Rel8/Type/Builder/Time.hs

```diff
@@ -0,0 +1,147 @@
+{-# language BangPatterns #-}
+{-# language NumericUnderscores #-}
+{-# language OverloadedStrings #-}
+{-# language PatternSynonyms #-}
+{-# language PartialTypeSignatures #-}
+{-# language TypeApplications #-}
+{-# language ViewPatterns #-}
+
+{-# options_ghc -Wno-partial-type-signatures #-}
+{-# options_ghc -Wno-unused-top-binds #-}
```

Comment:
```suggestion
```
We should drop these before merging

Reply:
No, we shouldn't. It's not possible to write a complete type signature for `divide` because `bytestring` does not export its `Monoidal` constraint. The `unused-top-binds` is because I don't actually use `YMD` from the pattern synonym, but I do use `ymdYear,`, `ymdMonth` and `ymdDay`.

Reply:
I see then add this as a comment